README.es-ES.md

```
@@ -0,0 +1,171 @@
+
+
+<div align="center">
+  
+  # <img src="logo.png" alt="Logo" height="32" style="vertical-align: middle;"> Cinemana Downloader Pro
```

markdownlint-cli2 (0.23.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)